raft/broadcast_tables/experimental/lang.hh

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+#pragma once
+
+#include <optional>
+#include <variant>
+
+#include "bytes.hh"
+
+
+namespace raft::broadcast_tables {
+
+using namespace std::string_literals;
+
+// Represents 'SELECT value WHERE key = {key} FROM system.broadcast_kv_store;'.
+struct select_query {
+    bytes key;
+};
+
+// Represents 'UPDATE system.broadcast_kv_store SET value = {new_value} WHERE key = {key} [IF value = {value_condition}];'.
+// If value_condition is nullopt, the update is unconditional.
+struct update_query {
+    bytes key;
+    bytes new_value;
```

Comment:
managed_bytes, so we can support large values

Reply:
Recall that we agreed in the office that we can go with `bytes` for now. Switching to `managed_bytes` is complex because there is no straightforward way to serialize them. We can try to do it as a follow-up, but we'll switch to a passive replication approach eventually anyway so I think `bytes` can stay for initial testing.